<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionInfo</class>
 <widget class="QDialog" name="QuestionInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>标题:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>811</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>提问者:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>281</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>110</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);
border-radius:8px;
width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>关注提问者</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>101</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>630</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);
border-radius:8px;
width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>赞</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>370</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);
border-radius:8px;
width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>关注回答者</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>890</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);
border-radius:8px;
width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>回复</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>问题ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>311</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>详情:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>730</y>
     <width>711</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>680</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>快捷回复:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>711</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 16pt &quot;华文行楷&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>回答者:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>580</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>回复时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>579</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>630</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>总赞数:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>630</y>
     <width>241</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>回复目录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>380</y>
     <width>81</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>回复ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>380</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>851</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 16pt &quot;华文行楷&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>890</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);
border-radius:8px;
width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>-10</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 32pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>问题详情</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);
border-radius:8px;
width:71px;
 height:22px;


</string>
   </property>
   <property name="text">
    <string>邀请回答</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
